services/github/src/tc-yaml.js

```diff
@@ -234,7 +234,7 @@ class VersionOne extends TcYaml {
   }
 
   compileTasks(config, cfg, payload, now) {
-    if (config.tasks.length > 0) {
+    if (config.tasks && config.tasks.length > 0) {
```

Comment:
Should it be `||`?

Reply:
Why?

Reply:
I must confess I didn’t quite understand from the description what was the bug (edited) 
It seems to me that change guards against an error which would be thrown in case config.tasks is undefined, but that’s not what you’re talking about in the PR description (or maybe I am misunderstanding)

Reply:
Sorry if the description was unclear -- that's exactly what I'm guarding against.  So we should treat the situation when `config.tasks` is `undefined` the same as `[]`.  I think this conditional is correct:
 * `undefined` -> `false && ..` -> false
 * `[]` -> true && false -> false
 * `[{some-task}]` -> true && true -> true

Reply:
got it!